src/pages/api/tutorials-map.ts

```diff
@@ -0,0 +1,35 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { getTutorialSlug } from 'views/collection-view/helpers'
+import { getAllTutorials } from 'lib/learn-client/api/tutorial'
+import moize, { Options } from 'moize'
+
+export default async function tutorialsMapHandler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const mapData = await cachedGenerateTutorialMap()
+  res.status(200).json(mapData)
```

Comment:
I can make this response behavior more robust, I just want to validate this approach first.